app/uk/gov/hmrc/helptosavefrontend/views/email/give_email.scala.html

```diff
@@ -21,7 +21,7 @@
 @import uk.gov.hmrc.helptosavefrontend.config.FrontendAppConfig
 @import uk.gov.hmrc.helptosavefrontend.views.helpers.errorPrefix
 
-@(form: Form[GiveEmail], backLink: String = "")(implicit htsContext: HtsContext, request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
+@(form: Form[GiveEmail], backLink: Option[String] = None)(implicit htsContext: HtsContext, request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
```

Comment:
are these optional because of the account holder change email journey? Are there no back links defined for that journey?

Reply:
yeah, not account holders but for DE users -  
for DE users there wont be any back link